src/components/dropdown/__snapshots__/Dropdown.spec.js.snap

```diff
@@ -2,10 +2,14 @@
 
 exports[`BDropdown render correctly 1`] = `
 <div class="dropdown dropdown-menu-animation is-mobile-modal">
-    <div role="button" tabindex="0" aria-haspopup="true" class="dropdown-trigger"><button class="trigger">trigger</button></div>
-    <div aria-hidden="true" class="background" style="display: none;" name="fade"></div>
-    <div aria-hidden="true" class="dropdown-menu" style="display: none;" name="fade">
-        <div class="dropdown-content"></div>
+  <div tabindex="0" class="dropdown-trigger" aria-haspopup="true"><button class="trigger">trigger</button></div>
+  <transition-stub name="fade" appear="false" persisted="false" css="true">
+    <div class="background" aria-hidden="true" style="display: none;"></div>
+  </transition-stub>
+  <transition-stub name="fade" appear="false" persisted="true" css="true">
+    <div class="dropdown-menu" aria-hidden="true" style="display: none;">
+      <div class="dropdown-content" aria-modal="true"></div>
```

Comment:
Extra `aria-modal="true"`?

Reply:
This is OK, because `aria-modal` attribute was introduced at 961bb82 after the last snapshot had been taken.